libraries/bxdf/standard_surface.mtlx

```diff
@@ -64,9 +64,9 @@
       <input name="in2" type="float" interfacename="coat_roughness" />
     </multiply>
     <mix name="coat_affected_rougness" type="float">
-      <input name="fg" type="float" interfacename="specular_roughness" />
-      <input name="bg" type="float" nodename="coat_affect_roughness_multiply2" />
-      <input name="mix" type="float" value="1.0" />
+      <input name="fg" type="float" value="1.0" />
```

Comment:
Just checking here as you swapped all 3 terms. I assume that it correct now and the mixing comes from upstream.

Reply:
Yes, this is the equation for modulating roughness in https://autodesk.github.io/standard-surface/ (section 3.2)